tests/sentry/models/test_useroption.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+from sentry.models import UserOption, UserOptionValue
+from sentry.testutils import TestCase
+
+
+class UserOptionManagerTest(TestCase):
+    def test_simple(self):
+        self.user = self.create_user('foo@example.com')
+        self.org = self.create_organization(owner=None)
+        self.team = self.create_team(organization=self.org)
+        self.create_member(user=self.user, organization=self.org, teams=[self.team])
+
+        UserOption.objects.set_value(
+            user=self.user,
+            organization=self.org,
+            key='deploy-emails',
+            value=UserOptionValue.all_deploys,
+        )
```

Comment:
You should add an assert here to make sure that the `set_value` worked correctly.